Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -87,30 +112,34 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         [Header("Hand Simulation")]
         [SerializeField]
         [Tooltip("Enable hand simulation")]
-        private HandSimulationMode handSimulationMode = HandSimulationMode.Articulated;
-        public HandSimulationMode HandSimulationMode { get => handSimulationMode; set => handSimulationMode = value; }
+        private HandSimulationMode defaultHandSimulationMode = HandSimulationMode.Articulated;
```

Comment:
>defaultHandSimulationMode [](start = 35, length = 25)

Is it possible to FormerlySerializedAs here? It looks like this still reasonably equivalent.